usecases/MinimumWorkingExample/dodo.py

```diff
@@ -13,67 +10,109 @@
     .emodul_generate_processed_data import processed_data_from_rawdata
 
+from doit import get_var
+
+# set a variable to define a cheap or full run
+config = {"mode": get_var('mode', 'cheap')}
 
 DOIT_CONFIG = {'verbosity': 2}
+
 #parent directory of the minimum working example
 ParentDir = os.path.dirname(Path(__file__))
 
-#raw data files is a subdirectory
-raw_data_emodulus_directory = Path(ParentDir, 'Data', 'E-modul')
-#meta data is in a different subdirectory
-metadata_emodulus_directory = Path(ParentDir, 'emodul',
-                                    'metadata_yaml_files')
-#processed data is in a different subdirectory
-processed_data_emodulus_directory = Path(ParentDir, 'emodul',
-                                    'processed_data')
+# defining paths
+emodul_output_directory = Path(ParentDir, 'emodul')  # folder with metadata yaml files
+raw_data_emodulus_directory = Path(ParentDir, 'Data', 'E-modul')  # folder with folders of raw data files
+metadata_emodulus_directory = Path(emodul_output_directory, 'metadata_yaml_files')  # folder with metadata yaml files
+processed_data_emodulus_directory = Path(emodul_output_directory, 'processed_data')  # folder with csv data files
+knowledge_graphs_directory = Path(emodul_output_directory, 'knowledge_graphs')  # folder with KG ttl files
 
-knowledge_graphs_directory = Path(ParentDir, 'emodul', 'knowledge_graphs')
+# when "cheap option" is run, only this souce of raw data is processed
+cheap_example_name = 'Wolf 8.2 Probe 1'
+
+# create folder, if it is not there
+Path(emodul_output_directory).mkdir(parents=True, exist_ok=True)
 
 #extract standardized meta data for Young' modulus tests
 def task_extract_metadata_emodul():
-    for f in os.scandir(raw_data_emodulus_directory):
+    # create folder, if it is not there
+    Path(metadata_emodulus_directory).mkdir(parents=True, exist_ok=True)
+
+    # setting for fast test, defining the list
+    if config['mode'] == 'cheap':
+        list_raw_data_emodulus_directories = [ Path(raw_data_emodulus_directory, cheap_example_name) ]
+    else: # go through all files
+        list_raw_data_emodulus_directories = os.scandir(raw_data_emodulus_directory)
+
+    for f in list_raw_data_emodulus_directories:
         if f.is_dir():
             raw_data_path = Path(f)
             raw_data_file = Path(f, 'specimen.dat')
             yaml_metadata_file = Path(metadata_emodulus_directory, f.name + '.yaml')
             yield {
                 'name': yaml_metadata_file,
-                'actions': [(emodul_metadata, [raw_data_path,
-                                               yaml_metadata_file])],
+                'actions': [(emodul_metadata, [raw_data_path, yaml_metadata_file])],
                 'file_dep': [raw_data_file],
-                'targets': [yaml_metadata_file],
-                'clean': [clean_targets]
+                'targets': [yaml_metadata_file]
+                #'clean': [clean_targets]  # what does this do?
```

Comment:
this defines if you specifit doit clean what is supposed to be deleted, i.e. all the tmp files are deleted and if done correctly you would start from scratch. I think this should be kept.